tests/comm/packet_check.cpp

```diff
@@ -0,0 +1,33 @@
+//
+// Created by Heather D'Souza on 2017-11-29.
+//
+
+#include "comm/PacketConstructor.h"
+#include "../../include/gtest-1.8.0/include/gtest/gtest.h"
+
+using namespace wlp;
+
+class TestPacket : public PacketConstructor {
+public:
+    TestPacket(const float* p_data, const Bitset<2> &packetType, const Bitset<6> &packetName) : PacketConstructor(p_data, packetType, packetName) {}
+
+    void getFromParent(float *data) {
+        get(data);
+    }
+};
+
+float *data = new float [3];
+Bitset<2> type1;
+Bitset<6> name;
+Bitset<1> test;
+
+TEST(packet_test, test_getBitsetNum){
```

Comment:
These tests are looking a little weak. I'd like to see at least a full test which sets start and end bits, packet name, packet type, and three data values (that aren't just zero), and confirms that they are properly set.